src/schema_generators/generators.cljc

```diff
@@ -198,21 +206,24 @@
    constraints is an optional mapping from schema to wrappers for the default generators,
    which can impose constraints, fix certain values, etc."
   ([schema]
-     (generator schema {}))
-  ([schema leaf-generators]
-     (generator schema leaf-generators {}))
+   (generator schema {:max-retries 10}))
+  ([schema opts]
```

Comment:
This is a breaking change, the new parameter needs to come last.

Reply:
Good point. I did this change make usually the function `generator` is being used without the extra arguments 🤔, which means you need to pass all arguments to eventually pass the options.

I will think on another approach.

Reply:
Yeah it's true, in retrospect maybe these should have been keyword args.  But I don't think it's that big a deal to have to pass a {} {} to use this new option ?

Reply:
I think all of these parameters are options, they could be on a keyword args as you suggested. Let us stick with what we have ;)

Reply:
Changes on https://github.com/plumatic/schema-generators/pull/29/commits/7043572018c08c519a1fa712d9f3ef7834acb3b6 and https://github.com/plumatic/schema-generators/pull/29/commits/3bc12dcfb35437788737f4b5dbf311d805fd4e2f